ui/src/components/ripple-out/RippleOutStoryContainer.js

```diff
@@ -33,111 +108,62 @@ const RippleOutStoryContainer = () => {
           h="100%"
           textStyle="darker_grotesque"
         >
-          <Flex flexDir="column" gap={9}></Flex>
-          <VStack gap="4em" textAlign="center">
+          <Text fontSize={headerFontSize} fontWeight={800}>
+            THE STORY
+          </Text>
+          <VStack mt={['2.5em', '5em']} gap="4em" textAlign="center">
             <Text
-              fontSize={['3em', '4.75em']}
-              lineHeight="1em"
+              fontSize={['2.25em', '2.75em', '3.25em', '3.75em', '4.25em']}
+              lineHeight={['1.5em', '1.5em']}
               textStyle="darker_grotesque_medium"
             >
-              We are called to <u>RIPPLE OUT</u> as we live out our{' '}
-              <u>SATURATE VISION</u> to saturate Hong Kong with the knowledge of
-              God’s glory
-            </Text>
-            <VStack lineHeight="2em">
-              <Text fontSize={['3xl', '4xl']}>
-                TRANSFORMING LIVES TRANSFORMING THE WORLD
-              </Text>
-              <Text
-                fontSize={['3xl', '4xl']}
-                textStyle="darker_grotesque_medium"
+              <Highlight
+                query={['ripple out', 'saturate vision']}
+                styles={{
+                  px: '1',
+                  bg: '#D7BB84',
+                  fontWeight: '800',
+                  color: '#182E57',
+                }}
               >
-                Transformation Center {'>>'} Kwun Tong {'>>'} Hong Kong {'>>'}{' '}
-                Ends of the earth
-              </Text>
-            </VStack>
-
-            <Button
-              variant="outline"
-              onClick={onOpen}
-              fontSize="lg"
-              bgColor="#ffffff"
-              fontWeight="bold"
-              borderColor="#182E57"
-              color="#182E57"
+                We are called to ripple out as we live out our Saturate vision
+                to saturate Hong Kong with the knowledge of God’s glory
+              </Highlight>
+            </Text>
+            <Image
+              src={
+                process.env.PUBLIC_URL +
+                '/images/ripple-out/ripple-out-story-circles.png'
+              }
+              boxSize={'100%'}
+              height={['11.5em', '13em', '15em', '20em', '100%']}
+              objectFit="cover"
+            />
+            <Text
+              fontSize={['1.5em', '1.75em', '2.25em', '2.5em', '2.75em']}
+              fontWeight={800}
+              textStyle="darker_grotesque_medium"
             >
-              READ THE STORY
-            </Button>
+              TRANSFORMING LIVES. TRANSFORMING THE WORLD.
+            </Text>
+            <Text
+              fontSize={['1.65em', '1.80em', '2.40em', '2.65em', '2.9em']}
+              fontWeight={600}
+            >
+              SWIPE TO READ OUR STORY {'>>'}
+            </Text>
+            <Slider {...sliderSettings} style={sliderStyle}>
```

Comment:
@Midnite1212 can steal from this for swipe behavior

Reply:
but also, missing left/right arrows from the design?

<img width="992" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/16095743/78a70114-eb9a-44fe-864c-e0a929702f3c">

Reply:
eh the arrows should only disappear on mobile and iPad mini view... did you start off with desktop view?

Reply:
oh so the arrows aren't reactive? like if i switch over from desktop to mobile to desktop viewports it affects the arrow visibillity?